cpp/URLHostObjects.h

```diff
@@ -3,10 +3,12 @@
 
 #include "utils.h"
 
-namespace fasturl {
+namespace fasturl
```

Comment:
My editor auto-formats c++ code - I can try to undo this if you dislike these changes